gemup.js

```diff
@@ -40,7 +40,7 @@
         var geminiKey = res.policy_document.conditions[1][2]
         var bucket = res.policy_document.conditions[0].bucket
         var data = {
-          'Content-Type': 'image/png',
+          'Content-Type': file.type,
```

Comment:
Hm interesting - so this change would basically just mean that we're passing the correct content type for the file to s3 instead of asserting that it's an `image/png`? It wouldn't have an impact on performance/file type/storage but would just be...more correct?

Reply:
Right. Seems like it doesn't have much of an impact with the way we're processing images today. I assume this has been able to take other file types just fine.

Just seems like we should respect the type of the file instead of hard-coding this?